Tools/Replay/Replay.h

```diff
@@ -69,7 +69,7 @@ class ReplayVehicle {
     AP_InertialNav_NavEKF inertial_nav{ahrs};
     AP_Vehicle::FixedWing aparm;
     AP_Airspeed airspeed;
-    DataFlash_Class dataflash{"Replay v0.1"};
+    DataFlash_Class dataflash{"Replay v0.1", 0}; // logging is magic for Replay
```

Comment:
How does this work? You should be passing an AP_Int32 reference. And wouldn't 0 prevent any logging at all?